packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoHelpers.kt

```diff
@@ -30,10 +31,10 @@ public object AndroidInfoHelpers {
   private fun isRunningOnStockEmulator(): Boolean =
       Build.FINGERPRINT.contains("generic") || Build.FINGERPRINT.startsWith("google/sdk_gphone")
 
-  @JvmStatic public fun getServerHost(port: Int): String = getServerIpAddress(port)
+  @JvmStatic public fun getServerHost(context: Context, port: Int): String = getServerIpAddress(context, port)
```

Comment:
This is a breaking change. It has to be an overload (or a second parameter + `@JvmOverload`).

Reply:
I returned support for `getServerHost(port: Int)` and in that case, when we have no context, I return `DEVICE_LOCALHOST` from `getServerIpAddress` (like in the previous implementation)